### DISCUSSION TITLE
Maintaining hover of tooltips and popover components

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#6777] [Q&A] [answered]
After setting the popover and tooltip, I want the popover to remain the same when I move the mouse cursor to the message. Is this possible?

```
import { Tooltip, Button } from '@mantine/core';

function Demo() {
  return (
    <Tooltip label="Tooltip">
      <Button>Button with tooltip</Button>
    </Tooltip>
  );
}
```

## example

![스크린샷 2024-09-05 오후 5 02 10](https://github.com/user-attachments/assets/502cc8f0-730e-481c-8f0f-751e9e6dca22)

**Comment by ryuhangyeong** [comment]
Is there any way within tooltip other than hover-card?

**Reply by rtivital** [reply, marked as answer]
You can fully control tooltip opened state with `opened` prop: `<Tooltip opened={yourCondition} />`.
